torch/csrc/jit/codegen/cuda/lower_allocation.cpp

```diff
@@ -113,10 +112,12 @@ class AllocationInserter : public kir::MutableIrVisitor {
       if (info.buffer->fuserTv()->axis(axis_i)->isReduction()) {
         continue;
       }
-      auto ca_id =
-          gpu_lower->lowerValue(fuser_tv->getComputeAtAxis(axis_i).first)
+      auto concrete_id =
+          gpu_lower
+              ->lowerValue(gpu_lower->caParallelMap().getConcreteMappedID(
+                  fuser_tv->axis(axis_i)))
               ->as<kir::IterDomain>();
-      init_dims.push_back(ca_id);
+      init_dims.push_back(concrete_id);
     }
```

Comment:
TODO: make sure this comes through https://github.com/csarofeen/pytorch/pull/601